src/Table/TablePagination.js

```diff
@@ -138,12 +140,17 @@ class TablePagination extends React.Component {
             })}
           </Typography>
           <div className={classes.actions}>
-            <IconButton onClick={this.handleBackButtonClick} disabled={page === 0}>
+            <IconButton
+              onClick={this.handleBackButtonClick}
+              disabled={page === 0}
+              {...IconButtonPreviousProps}
```

Comment:
I see that we use the *back* wording. Would that make more sense than *previous*?

Reply:
Yeah, not a big deal. Let me know what you guys are more comfortable with

Reply:
Let's go for `back`. We are using this wording on the MobileSteper. Better be consistent.

Reply:
Also, should it be `backIconButtonProps` or `IconButtonBackProps`?

Reply:
My preference would be for the first one. But I'm eager to hear what people think.

Reply:
I like the first one `backIconButtonProps`. If we're looking at MobileStepper using camelcase too it makes sense to stick with it.

Reply:
Updated